### PR TITLE
interfaces/hardware-observe: clean up reading access to sysfs

### PR DESCRIPTION
**Pull request by bugraaydogar**
Hi,

The hardware-observe interface already defines the following AppArmor rule `/sys/{block,bus,class,devices,firmware}/{,**} r,`. Therefore, explicit definition of further access requests for specific sysfs paths are not needed and creates confusion. Therefore, this PR aims to remove all unnecessary explicit sysfs paths.

Thanks

### CODE REVIEW

**Review by mardy — APPROVED** [review]
Thanks, LGTM! Since we are only removing roles I don't see the need of involving the security team, but let's see if others disagree.

**Review by pedronis — APPROVED** [review]
looks correct

**Comment by pedronis** [comment]
> Thanks, LGTM! Since we are only removing roles I don't see the need of involving the security team, but let's see if others disagree.

I don't think we should block on a review, but we should ping @alexmurray for awareness

**Review by alexmurray — APPROVED** [review]
LGTM - I wonder if it would be useful to add some tests for this though to ensure we don't inadvertently regress these access if changes are made to that more general AppArmor rule?

**Pull request merged**
